### PR TITLE
[MRG]:  re-enable reading of legacy KIT files

### PR DESCRIPTION
**Pull request by christianbrodbeck**
WIP addressing #4634 

I hard coded the parameters for NY, @kingjr can you verify that it reads the file you need correctly?

### CODE REVIEW

**Comment by larsoner** [comment]
@kingjr do you have time to look? This is one of the last remaining blockers for 0.15

File: mne/io/kit/kit.py [hunk 1]
```diff
@@ -26,12 +26,20 @@
...
 from ..constants import FIFF
 from ..meas_info import _empty_info, _read_dig_points, _make_dig_points
-from .constants import KIT
+from .constants import KIT, LEGACY_AMP_PARAMS
 from .coreg import read_mrk
 from ...externals.six import string_types
 from ...event import read_events
 
 
+class UnsupportedKITFormat(Exception):
```
Comment (larsoner) [inline mne/io/kit/kit.py]: To be consistent with the MaxFilter error in FIFF this should probably subclass `ValueError`

**Comment by larsoner** [comment]
@christianbrodbeck this is ready to go from your end, right?

**Comment by christianbrodbeck** [comment]
Yes from my end it's ready

**Comment by agramfort** [comment]
thx @christianbrodbeck

**Pull request merged**